Project_3/style.css

```diff
@@ -0,0 +1,240 @@
+/* ==============
+OVERALL DESIGN
+================ */
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #e5eeea;
+}
+
+/* ===========
+HEADER
+============== */
+
+header {
+    background-color: #ffffff;
+    height: 100vh;
+    margin-bottom: 100px;
+    padding: 15%;
+    width: 100%;
```

Comment:
The `header` element has 100% width by default, so you should be able to remove this.
```suggestion
```